swagger.json

```diff
@@ -2321,7 +2308,7 @@
             },
             "delete": {
                 "tags": [
-                    "Default"
+                    "Fleet", "Industrial"
```

Comment:
Did you intend to add both Fleet and Industrial to these tags endpoints? Maybe we should create a separate Tags tag?

Reply:
I assumed it would be easiest for Industrial customers to easily go to one spot to find all the APIs that are relevant to industrial, assuming everything else is Fleet-only. Per second look, it seems that Sensors are also related to Industrial and not in the Industrial section, so I'll similarly create a separate Tags section.